gamemodes/terrortown/entities/entities/ttt_game_text.lua

```diff
@@ -66,7 +66,10 @@ function ENT:AcceptInput(name, activator)
 	elseif inputReceiver == RECEIVE_TRAITOR then
 		messageReceiver = GetTeamChatFilter(TEAM_TRAITOR)
 	elseif inputReceiver == RECEIVE_INNOCENT then
-		messageReceiver = GetTeamChatFilter(TEAM_INNOCENT)
+		messageReceiver = GetPlayerFilter(function(p)
+			local plyRoleData = ply:GetSubRoleData()
```

Comment:
Can you please add a small comment here explaining that reasoning? So it clears up confusion about why this does not only filter innocents